### PR TITLE
Add components for bookstore project

### PR DESCRIPTION
**Pull request by Jawad-Nazari**
**In this PR, I did the following:**

-  Created a new branch then I set up React and Linters on it ✅
-  Initialized the routes in App.js ✅
-  Created card, Header, and Layout components ✅
-  Created BooksList, Book, and NewBook components. ✅
-  Added a BookList, and NewBook components inside my BooksPage. ✅
-  Updated the readme file and also checked the linter errors.✅

### CODE REVIEW

File: src/components/NewBook.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Card from './UI/Card';
+
+const NewBook = () => (
+  <Card>
+    <h3>Add New Book</h3>
+    <form>
+      <div>
+        <input type="text" placeholder="Book Title" />
```
Comment (AdnanOlarmmi) [inline src/components/NewBook.js]: - [ ] Nice one adding an input option for `book` title, kindly note that
  - [ ] The `input` tag doesn't have the `name` attribute.
  - [ ] The `form` doesn't have an `author` input, which is also a requirement.

File: src/components/NewBook.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          <option value="Action">Action</option>
+          <option value="Romance">Romance</option>
+          <option value="Economy">Economy</option>
+          <option value="History">History</option>
+          <option value="Fantasy">Fantasy</option>
+          <option value="Science Fiction">Science Fiction</option>
+        </select>
+      </div>
+      <div>
+        <button type="button">Add Book</button>
```
Comment (AdnanOlarmmi) [inline src/components/NewBook.js]: - [ ] Following the DRY principle, this should be in an arrray then, you should map through the array, something like this.

const categories = ["Category", "Action", "Romance", "Economy", "History", "Fantasy", "Science Fiction"]

```
<div>
  <select>
    {categories.map((category)=> {
     <option value={category}>{category}</option>
})}
  </select>
</div>
```

Keeping ones code DRY is very very important.

**Review by AdnanOlarmmi — CHANGES_REQUESTED** [review]
Hi @Jawad-Nazari, 

This is Adnan Is-haq, your beloved code reviewer,

You've done a very good job so far, but a few things still need to be worked on before moving on to the next project, but you're so close!

You're Almost there, funny one eyed alien

## Highlights
- React-dom used properly.
- App rendered properly.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/NewBook.js (hunk 1 above)
Comment (Jawad-Nazari) [reply src/components/NewBook.js]: Hi @AdnanOlarmmi ,

Thanks for your time and for reviewing my code. I have updated my code as you advised.

File: src/components/NewBook.js (hunk 2 above)
Comment (Jawad-Nazari) [reply src/components/NewBook.js]: Hi @AdnanOlarmmi ,

Thanks, for your time and for reviewing my code, I have update my code as you advised. 🙏

**Review by mikethreels — APPROVED** [review]
Hi Jawad, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**